src/system/components/typography/Chip/style.scss

```diff
@@ -47,15 +48,19 @@
 
 .ds-chip-size-base {
   font-size: $font-size-small;
+  padding-left: $space-small * 0.9;
+  padding-right: $space-small * 0.9;
```

Comment:
The whole purpose of design tokens gets negated by using custom computations. Please just use $space-small instead.

Reply:
Yeah I’m also unhappy with it but I’d does not look right. I just miss some pixels to get it right. 🤔

Reply:
Maybe I need to work here with em values instead.

Reply:
That would be a viable option. Please have a look at the font-spacing tokens then.